systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -638,25 +641,25 @@ private String getExporterRunScript(String podName) throws InterruptedException,
     }
 
     private void assertCoMetricNotNull(String metric, String kind, HashMap<String, String> data) {
-        Pattern connectResponse = Pattern.compile(metric + "\\{kind=\"" + kind + "\",} ([\\d.][^\\n]+)");
+        Pattern connectResponse = Pattern.compile(metric + "\\{kind=\"" + kind + "\",} ([\\d.][^\\n]+)", Pattern.CASE_INSENSITIVE);
         ArrayList<Double> values = MetricsUtils.collectSpecificMetric(connectResponse, data);
         assertThat(values.stream().mapToDouble(i -> i).count(), notNullValue());
     }
 
     private void AssertCoMetricResourceStateNotExists(String kind, String name) {
```

Comment:
Can you please rename this to `assertCoMetricResourceStateNotExists`?

Reply:
Done